alibi/api/defaults.py

```diff
@@ -249,3 +249,20 @@
 """
 Default CounterfactualRL data.
 """
+
+
+DEFAULT_META_PROTOSELECT = {"name": None,
+                            "type": ["prototypes"],
+                            "explanation": ["global"],
```

Comment:
These aren't really consistent/required because the method is not an explainer, let's discuss.

Reply:
add some data terminology?